jenkins/jobs/integration_tests.pipeline

```diff
@@ -94,11 +96,26 @@ pipeline {
               echo "${err}"
               currentBuild.result = 'FAILURE'
           }
+          finally {
```

Comment:
what if job fails, will we still be able to run the final script to fetch the logs?

Reply:
yes, that's why there is a finally